Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
 name = "flux-lsp"
-version = "0.5.19"
-authors = ["Brandon Farmer <bthesorceror@gmail.com>"]
+version = "0.5.20"
+authors = [
+	"Sean Brickley <sbrickley@influxdata.com",
```

Comment:
Can this get changed to something that includes everyone in the flux team? I'm a little hesitant to keep adding contributors like this. I'd rather have a single common email.

Reply:
That's fine with me. Any suggestions for what we should change it to?